lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -97,9 +97,7 @@ <h3 class="hd hd-3">${_("Reports Available for Download")}</h3>
 
     ## Translators: a table of URL links to report files appears after this sentence.
     <p>
-        ${Text(_("{note_start}: To keep student data secure, you cannot save or email these links for direct access. Copies of links expire within 5 minutes.")).format(
-            note_start=HTML("<strong>Note</strong>")
-        )}
+        ${_("<strong>Note</strong>: To keep student data secure, you cannot save or email these links for direct access. Copies of links expire within 5 minutes.")}
```

Comment:
This seems like it shouldn't be working.  As written, this should show the text `<strong>Note</strong>: ...` as-is on the screen. 

The original was closer, but had an i18n issue.  I'm not sure what you mean by "it wasn't working".  I imagine this code should look as follows:
```
${Text(_("{strong_start}Note{strong_end}: To keep student data secure, you cannot save or email these links for direct access. Copies of links expire within 5 minutes.")).format(
    strong_start=HTML("<strong>"),
    strong_end=HTML("</strong>"),
)}
```
But again, I'm not sure what issue you were seeing.